### DISCUSSION TITLE
Syntax error: unexpected token u in JSON at position 0 when connecting Postgre

### DISCUSSION THREAD

**Discussion by averrows** [Budibase/budibase#3486] [Help]
As I try to connect budibase with postgre (from digitalocean), I encounter this error. Configurations are not false as I can connect from other platform.
![image](https://user-images.githubusercontent.com/42243557/142967811-5197b5b2-7df2-419e-994f-0ad07e86b6a1.png)

**Comment by mjashanks** [comment]
Hi There.

Do you need to whitelist your digital ocean IP?

Does this error return immediately, or does it take 10seconds or more?

**Reply by averrows** [reply]
It doesn't not return immediately, there is time gap between clicking the button and the error message.

**Reply by mjashanks** [reply]
That would suggest that the connection is timing out. I.e. 
- your budibase deployment cannot reach the database. 
- your database is running very slowly
- your database has an enormous amount of tables (budibase runs a query to fetch tables)

It would be useful if you could verify whether a connection is being made, from the database side.

**Reply by satriowicaksn** [reply]
Hi @mjashanks , if we want to whitelist IP in digital ocean, where we can see our cloud budibase IP?

**Reply by shogunpurple** [reply]
Info here: https://docs.budibase.com/building-apps/data#getting-started

`34.246.43.190`

**Reply by satriowicaksn** [reply]
ok big thanks @shogunpurple @mjashanks , i will try to whitelist this IP
